common/web/types/src/main.ts

```diff
@@ -28,3 +28,5 @@ export { TouchLayoutFileWriter, TouchLayoutFileWriterOptions } from './keyman-to
 export * as KPJ from './kpj/kpj-file.js';
 export { KPJFileReader } from './kpj/kpj-file-reader.js';
 export { KeymanDeveloperProject } from './kpj/keyman-developer-project.js';
+
+export * as util from './util/util.js';
```

Comment:
would be clearer as `fs-util.ts` or `path-util.ts` in light of the encapsulation mentioned.

Reply:
Note that follow-up PR #8674 refactors the fs-related functions in util.ts away again, moving responsibility for path and fs ops into the callbacks (we may be able to do better later but avoiding adding extra deps for now). :grin: